website/server/controllers/api-v3/chat.js

```diff
@@ -207,8 +207,8 @@ api.likeChat = {
  *
  * @apiUse GroupNotFound
  * @apiUse MessageNotFound
- * @apiError FlagOwnMessage Chat messages cannot be flagged by the author of the message
- * @apiError AlreadyFlagged Chat messages cannot be flagged more than once by a user
+ * @apiError (404) {NotFound} FlagOwnMessage Chat messages cannot be flagged by the author of the message
+ * @apiError (404) {NotFound} AlreadyFlagged Chat messages cannot be flagged more than once by a user
```

Comment:
FlagOwnMessage shouldn't exist. It is possible (deliberately) for the author to flag their own message.